seed-job/parser.py

```diff
@@ -22,7 +23,10 @@
         "-p TARGET_FILE={} ".format(entry["target-file"]) + \
         "-p CONTEXT_DIR={} ".format("master-job") + \
         "-p APP_ID={} ".format(entry["app-id"]) + \
-        "-p JOB_ID={} ".format(entry["job-id"])
+        "-p JOB_ID={} ".format(entry["job-id"]) + \
+        "-p REGISTRY_URL={}".format(
+            os.environ.get("REGISTRY_URL") or "registry.centos.org"
```

Comment:
It's okay to set this as an env var for now. But I'd like us to have a config file where we set values. Something like we do with `hosts` file in current service. 

Also, I don't want to be making any assumptions and hard-coding. It has only brought problems to our service. So we might even want to remove the `or "registry.centos.org"` part. Deployment **should** fail if the value for `REGISTRY_URL` is not found. 

WDYT @bamachrn ?